modules/lib.lua

```diff
@@ -268,6 +270,11 @@ if isServer then
 
         if ped then
             SetPedIntoVehicle(ped, veh, -1)
+            bucket = GetEntityRoutingBucket(ped) or nil
+        end
+
+        if bucket then
+            SetEntityRoutingBucket(veh, bucket)
```

Comment:
Add an additional check to make sure that the bucket is not "0" no reason to set the bucket to 0 when its by default spawned at 0.

also the native has been fixed using oal

Reply:
```suggestion
            bucket = GetEntityRoutingBucket(ped) or nil
        end

        if bucket and bucket > 0 then
            SetEntityRoutingBucket(veh, bucket)
```